README.md

```diff
@@ -18,11 +18,14 @@ template for _instances_.
 
 ### Write a Method That Uses Instance Data and Parameter Data
 
-- `Cat`, `Dog`, and `Bird` classes accept parameters: `name`,`sex`
-- If `sex` of `Bird` is "male", variable `canTalk` equals true
-- Write function `speak`: `Cat` says "`name` says meow!", `Dog` says "`name`
-says woof!" and `Bird`, if male, says "It's me! `name`, the parrot!", if not,
-it says "Squawk!".
+To further explore _OOP_ concepts, let's create 3 classes that will contain
+`constructors` and functions that leverage _parameter_ data. In `index.js`,
```

Comment:
Inside a class, I've been using the term is `method` rather than function to be consistent with MDN documentation.